README.md

```diff
@@ -19,3 +19,12 @@ migrate -url postgres://user@host:port/database -path ./db/migrations -timeout 1
 migrate -url postgres://user@host:port/database -path ./db/migrations -timeout 10 down 1
 migrate help # for more info
 ```
+
+## How to contribute
```

Comment:
I propose a bit of formatting (and a typo fix):

```markdown
1. Fork the repo on Github.
2. Clone the `wallester/migrate` repo. Next steps are to be done in the `wallester/migrate repo` (not the fork).
3. Add your fork as a new remote (`git remote add myfork url-of-myfork`).
4. Create a new branch, do your work and commit the changes as usual.
5. Push your new branch to your fork (`git push myfork mybranch`).
6. Open a pull request in Github as usual.
```

Reply:
thanks, changes applied